src/python/turicreate/toolkits/drawing_classifier/drawing_classifier.py

```diff
@@ -1069,8 +1069,9 @@ class as a vector. Label ordering is dictated by the ``classes``
             Rows: 10
             [3, 4, 3, 3, 4, 5, 8, 8, 8, 4]
         """
+        feature_column = self.__proxy__.get_state()["feature_column_name"]
```

Comment:
How would we access a private member variable from the Python layer?

Reply:
I believe this can just be `self.feature_column_name` without any changes elsewhere, due to the magic of Python and `ExposeAttributesFromProxy` (https://github.com/apple/turicreate/blob/209c8a64be2c8aea6c278acc2f15bb5790d88d8b/src/python/turicreate/toolkits/_model.py#L308)